tracing-bom/build.gradle

```diff
@@ -42,11 +42,13 @@ micronautBom {
                         "io.zipkin.proto3"
                 ] as Set
         )
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.30.0")
+        dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.30.0-alpha")
         dependencies.add("io.zipkin.brave:brave-bom:5.16.0")
         dependencies.add("io.zipkin.reporter2:zipkin-reporter-bom:2.16.3")
-        dependencies.add("io.opentelemetry:opentelemetry-bom:1.26.0")
-        dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.26.0-alpha")
-        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.26.0")
-        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.26.0-alpha")
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.29.0")
```

Comment:
this should 1.30.1 https://repo1.maven.org/maven2/io/opentelemetry/opentelemetry-bom/1.30.1/

Reply:
@sdelamo Sorry, we can't do that.

Please note that `io.opentelemetry:opentelemetry-bom:1.30.1` is already added as a dependency in line 45. We just need to add `1.29.0` as well, because [`opentelemetry-instrumentation-bom`](https://central.sonatype.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-bom) is still on version 1.29.0 and it depends on that version of the BOM.

If I try to remove these 1.29.0 dependencies, then `checkBom` fails with:

```
BOM io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.29.0
  (via io.micronaut.tracing:micronaut-tracing-bom:5.1.0-SNAPSHOT
    -> io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.29.0)
  is not a Micronaut BOM but it imports another BOM
  (io.opentelemetry:opentelemetry-bom:1.29.0)

BOM io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha
  (via io.micronaut.tracing:micronaut-tracing-bom:5.1.0-SNAPSHOT
    -> io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha)
  is not a Micronaut BOM but it imports another BOM
  (io.opentelemetry:opentelemetry-bom:1.29.0)

BOM io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha
  (via io.micronaut.tracing:micronaut-tracing-bom:5.1.0-SNAPSHOT
    -> io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha)
  is not a Micronaut BOM but it imports another BOM
  (io.opentelemetry:opentelemetry-bom-alpha:1.29.0-alpha)

BOM io.opentelemetry:opentelemetry-bom-alpha:1.29.0-alpha
  (via io.micronaut.tracing:micronaut-tracing-bom:5.1.0-SNAPSHOT
    -> io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha
      -> io.opentelemetry:opentelemetry-bom-alpha:1.29.0-alpha)
  is not a Micronaut BOM but it imports another BOM
  (io.opentelemetry:opentelemetry-bom:1.29.0)
```

Reply:
@melix what do you think we should do?